package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@manuscripts/json-schema": "^2.0.0",
+    "@manuscripts/data": "1.0.1",
```

Comment:
I believe that there is a 1.0.2 version of data, so let's use that. Also, you should have a caret there.

Reply:
Actually, we already have https://github.com/Atypon-OpenSource/manuscripts-transform/pull/18 which updates manuscripts-data to 1.0.2. We should just wait for that.